### PR TITLE
fix(edgeless): toolbar right-side clipping position

### PR DESCRIPTION
**Pull request by alpakaka0o0**
closes https://github.com/toeverything/blocksuite/issues/5449

Changes:

  The value of the _popperShow variable in the more-button is now passed to the component-toolbar. In the component-toolbar.ts, I modified the _computePosition() function to calculate the left based on the presence of the more-actions-container.
  
  
Before:
before

After:

https://github.com/toeverything/blocksuite/assets/115137708/a6979e11-3f95-4b82-adbb-a34d291c99df

### CODE REVIEW

File: packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts [hunk 1]
```diff
@@ -200,6 +203,11 @@ export class EdgelessComponentToolbar extends WithDisposable(LitElement) {
       : nothing;
   }
 
+  protected getPopperShow = (popperShow: boolean) => {
+    this._popperShow = popperShow;
```
Comment (doodlewind) [inline packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts:207]: You should not set a value inside a `getSomething` method, whose semantics should be read only.

**Review by doodlewind — COMMENTED** [review]
Great to see your contribution, could you [sign our CLA ](https://github.com/toeverything/blocksuite/edit/master/.github/CLA.md) for the following review process? Thanks!

File: packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts (hunk 1 above)
Comment (alpakaka0o0) [reply packages/blocks/src/page-block/edgeless/components/component-toolbar/component-toolbar.ts:207]: Thank you for review my code. I will fix it quickly!

**Review by doodlewind — APPROVED** [review]
Thanks for the fix!

**Pull request merged**